plugins/modules/mso_user.py

```diff
@@ -248,14 +248,17 @@ def main():
         if mso.existing:
             if not issubset(mso.sent, mso.existing):
                 # NOTE: Since MSO always returns '******' as password, we need to assume a change
-                if 'password' in mso.proposed:
-                    mso.module.warn("A password change is assumed, as the MSO REST API does not return passwords we do not know.")
-                    mso.result['changed'] = True
+                mso.stdout = 'reached pwd' + '\n'
+                mso.module.warn("A password change is assumed, as the MSO REST API does not return passwords we do not know.")
+                mso.result['changed'] = True
 
                 if module.check_mode:
```

Comment:
This condition should probably be adjusted to make sure it only fires when needed.